port/jemalloc_helper.h

```diff
@@ -40,23 +40,34 @@ static inline bool HasJemalloc() { return true; }
 
 // Declare non-standard jemalloc APIs as weak symbols. We can null-check these
 // symbols to detect whether jemalloc is linked with the binary.
-extern "C" void* mallocx(size_t, int) __attribute__((__nothrow__, __weak__));
-extern "C" void* rallocx(void*, size_t, int) __attribute__((__nothrow__, __weak__));
-extern "C" size_t xallocx(void*, size_t, size_t, int) __attribute__((__nothrow__, __weak__));
-extern "C" size_t sallocx(const void*, int) __attribute__((__nothrow__, __weak__));
-extern "C" void dallocx(void*, int) __attribute__((__nothrow__, __weak__));
-extern "C" void sdallocx(void*, size_t, int) __attribute__((__nothrow__, __weak__));
-extern "C" size_t nallocx(size_t, int) __attribute__((__nothrow__, __weak__));
-extern "C" int mallctl(const char*, void*, size_t*, void*, size_t)
-    __attribute__((__nothrow__, __weak__));
-extern "C" int mallctlnametomib(const char*, size_t*, size_t*)
-    __attribute__((__nothrow__, __weak__));
-extern "C" int mallctlbymib(const size_t*, size_t, void*, size_t*, void*,
-                            size_t) __attribute__((__nothrow__, __weak__));
-extern "C" void malloc_stats_print(void (*)(void*, const char*), void*,
-                                   const char*) __attribute__((__nothrow__, __weak__));
-extern "C" size_t malloc_usable_size(JEMALLOC_USABLE_SIZE_CONST void*)
-    JEMALLOC_CXX_THROW __attribute__((__weak__));
+extern "C" JEMALLOC_ALLOCATOR JEMALLOC_RESTRICT_RETURN void JEMALLOC_NOTHROW *
+je_mallocx(size_t, int) JEMALLOC_ATTR(malloc) JEMALLOC_ALLOC_SIZE(1)
```

Comment:
Is there a reason this changed from `mallocx` to `je_mallocx`? `mallocx` is still referenced below in `HasJemalloc()`

Reply:
@pdillinger Sorry, that was unintentional. Should be fixed now.